frontend/src/components/App.js

```diff
@@ -6,6 +6,7 @@ import { CopCard, InnerCopCard } from "./Cards/Cards";
 import { ScrollCarousel } from "./Carousel/ScrollCarousel";
 import { InnerCopNav } from "./Navigation/InnerCopNav";
 import { HeaderNav } from "./Navigation/HeaderNav";
+import Footer from "../pages/Footer/_Footer";
```

Comment:
Let's put the Footer in components since it is not a full page.

Reply:
Reverted back to Navigation